editions/tw5.com/tiddlers/docker/Using TiddlyWiki with Docker.tid

```diff
@@ -0,0 +1,39 @@
+created: 20230403211230516
+modified: 20230403211230516
+tags: [[TiddlyWiki with Docker]]
+title: Using TiddlyWiki with Docker
+type: text/vnd.tiddlywiki
+
+~TiddlyWiki5 can also be run using Docker. See the [[Dockerfile]] for details.
```

Comment:
The file `Dockerfile` isn't part of the build process and so this link won't work. Perhaps we could make it:

```
[[Dockerfile|https://github.com/Jermolene/TiddlyWiki5/blob/master/Dockerfile]]
```

Reply:
I can do that!